src/base/base.pri

```diff
@@ -55,7 +55,12 @@ HEADERS += \
     $$PWD/torrentfileguard.h \
     $$PWD/torrentfilter.h \
     $$PWD/scanfoldersmodel.h \
-    $$PWD/searchengine.h
+    $$PWD/searchengine.h \
+    $$PWD/notifications/notifier.h \
+    $$PWD/notifications/dummynotifier.h \
+    $$PWD/notifications/notificationsmanager.h \
+    $$PWD/notifications/request.h \
```

Comment:
Don't use such common names for files. qmake compiles it in one directory so if we have `subfolder1/name.cpp` and `subfolder2/name.cpp` there is a conflict during compilation.